src/components/Plugin/SafeSnap/Form/RawTransaction.vue

```diff
@@ -113,24 +113,24 @@ export default {
 </template>
 
 <style lang="scss" scoped>
-.textarea {
-  border: 1px solid var(--border-color);
-  background-color: transparent;
-  color: var(--link-color);
-  border-radius: 23px;
-  padding: 0 24px;
-  outline: none;
-  font-size: 14px;
-
-  &:hover {
+  .textarea {
```

Comment:
Can you remove lint here? Otherwise this PR is ready to go

Reply:
fixed @bonustrack